build.gradle

```diff
@@ -107,6 +111,11 @@ dependencies {
 
     compile 'org.projectlombok:lombok:1.18.4'
 
+    compile 'org.springframework.boot:spring-boot-starter-data-mongodb'
+    compile 'org.apache.commons:commons-collections4:4.4'
+    compile 'org.springframework.boot:spring-boot-starter-actuator:2.1.7.RELEASE'
```

Comment:
@m24murray are we using `actuator` in the learner record?

Reply:
Not yet, probs will in future. Ill remove for now